packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/internal/SyncConfigurationImpl.kt

```diff
@@ -26,7 +26,8 @@ import io.realm.mongodb.SyncConfiguration
 import io.realm.mongodb.SyncException
 import io.realm.mongodb.SyncSession
 
-internal class SyncConfigurationImpl(
+// TODO Public as it is used from the io.realm.mongodb.sync.SyncConfiguration interface
+public class SyncConfigurationImpl(
```

Comment:
But is it considered public when we are just calling the constructor? This sounds a bit weird, because then everything would need to be public as there is a public entry point for all internal things.

Reply:
Good point, fixed